docs/platforms/android/integrations/jetpack-compose/index.mdx

```diff
@@ -130,7 +133,8 @@ fun LoginScreen() {
 
 <Note>
 
-If a `@Composable` doesn't have a `testTag` modifier applied, both breadcrumbs and transactions won't be captured because they can't be uniquely identified. To automatically generate tags, refer to the docs about our <PlatformLink to="/enhance-errors/kotlin-compiler-plugin/">Sentry Kotlin Compiler Plugin</PlatformLink>.
+If a `@Composable` doesn't have a `sentryTag` modifier applied, both breadcrumbs and transactions won't be captured because they can't be uniquely identified. 
```

Comment:
```suggestion
If a `@Composable` doesn't have a `.sentryTag` or a `.testTag` modifier applied, both breadcrumbs and transactions won't be captured because they can't be uniquely identified. 
```